autoscheduler/frontend/src/components/SchedulingPage/Schedule/InstructionsDialog/InstructionsDialog.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {
+  Card, CardActions, CardContent, Button, Typography, Fade,
+} from '@material-ui/core';
+import { useSelector } from 'react-redux';
+import * as styles from './InstructionsDialog.css';
+import { RootState } from '../../../../redux/reducer';
+
+const InstructionsDialog: React.FC = () => {
+  const [open, setOpen] = React.useState(true);
+  const hasCurrentAv = useSelector<RootState, boolean>(
+    (state) => state.availability.length > 0,
+  );
+  // can have 1 of 3 values:
+  // null -> user's first time on site
+  // ''   -> user has not yet made an av
+  // 'Y'  -> user has made an av or has manually dismissed dialog in the past
+  const hasPastAv = localStorage.getItem('has-created-availability');
```

Comment:
We should be able to just `return null` here if this is true, right? It would make renders very marginally faster since the dialog is no longer on the page, and clean up the logic later when you have to check if `hasPastAv`

Reply:
I tried it, but it looks ugly because currently, the `Fade` component will gradually fade out the dialog, whereas returning `null` makes the dialog disappear suddenly. Will keep as is.